1.1/spec/index.md

```diff
@@ -1373,3 +1389,10 @@ Registrations. S. Leonard. IETF. March 2016. URL:  <https://www.rfc-editor.org/r
 
 <span id="ref-rfc8141"/>**\[RFC8141]** Uniform Resource Names (URNs). P. Saint-Andre; J. Klensin. IETF. April 2017.
 Proposed Standard. URL: <https://www.rfc-editor.org/rfc/rfc8141>
+
+## Revision history
+
+| Version | Date | Description |
+| --- | --- | --- |
+| [v1.1.0](https://ocfl.io/1.1.0/spec/) | 7 October 2022 | First v1.1, [release notes](https://ocfl.io/news/#version-11-of-the-oxford-common-file-layout-ocfl-released), [change log](https://ocfl.io/1.1.0/spec/change-log.html) |
+| [v1.1.1](https://ocfl.io/1.1.1/spec/) | 8 November 2024 | Clarifications, [release notes](https://ocfl.io/news/#version-111-of-the-oxford-common-file-layout-ocfl-released), [change log](https://ocfl.io/1.1/spec/change-log.html) |
```

Comment:
(https://ocfl.io/1.1/spec/change-log.html)->(https://ocfl.io/1.1.1/spec/change-log.html)

Reply:
Fixed in https://github.com/OCFL/spec/pull/654/commits/22f993007a0db2ecc67f1118d29fc5cb7aeb0a7c